### ISSUE TITLE
Error message when syncing with Soundcloud

### ISSUE THREAD

**Issue by rtomasello** [erwinkramer/SoundCloud-Playlist-Sync#20]
I got the following error message when trying to sync my Playlist on Soundcloud:

Unable to read manifest to determine tracks to delete; exception: TagLib.CorruptFileException: MPEG audio header not found.at TagLib.File.Create(FileAbstraction abstraction, String mimetype, ReadStyle propertiesStyle)
at TagLib.Fire.Create(String path)
at Soundcloud_Playlist_Downloader.Utils.MetadataTaggingUtils.TagIt(Track song)
at Soundcloud_Playlist_Downloader.Utils.SyncUtils.AlalyseManifestTracks(List'1 allTracks, List'1 tracksToDownload) (no inner exceptions found)

Can you please investigate this potential new issue? Thank you in advance

**Comment by mcprat** [comment]
List or screenshot of your options in the program?

**Comment by rtomasello** [comment]
<img width="315" alt="capture" src="https://user-images.githubusercontent.com/24699269/27507701-5d52485e-58cc-11e7-89ef-d0bc1a0692be.PNG">

**Comment by erwinkramer** [comment]
Do you know what song it happens at?

**Comment by erwinkramer** [comment]
You can identify it by watching the Downl. progress view and check for the song starting with an [x] .

**Comment by rtomasello** [comment]
download progress is not displayed on the 'Downl. Progress' box, the error message appears while fetching tracks to download, download does not start

**Comment by mcprat** [comment]
@erwinkramer , does it tag after converting to mp3? Might have to do with that.

Either way, I highly recommend checking the "Exclude m4a, aac" checkboxes. What that does is it allows m4a and aac files to not be converted to the lesser mp3 container. It saves a lot of time for not really much benefit as almost every music player can play aac and m4a. Might also help with tagging the metadata?

I also recommend setting concurrency (the number of tracks to fetch/download at one time) to 1, especially if your internet connection is less than average. Your error seems a little rare, but most of these errors that occur before downloads even start do not occur if you attempt the sync several times. Eventually, you should get it. Have you tried syncing more than once? have you gotten any other errors or the same every time?

**Comment by rtomasello** [comment]
@mpratt14 thank you for your suggestions. I've unticked 'Choose high quality versions if available' in order not to convert .m4a and .aac to .mp3. In addition, I have set 'amount' of concurrency to 1 as suggested. I've also tried to sync multiple times however I still have the same issue.

**Comment by mcprat** [comment]
I meant for you to check 'Choose high quality versions if available' and then check the two in the box that appears under 'Exclude:'. If you uncheck all those then everything comes as mp3. 

I just realized your problem.... Under Soundcloud URL, it should just be the link to your profile, (https://www.soundcloud.com/rickscafe) without the '/likes' part. If that doesn't work then idk.

**Comment by rtomasello** [comment]
I got you, I've now flagged both 'Choose high quality versions if available' and the two checks in the box to exclude .m4a and .aac as per attached screenshot, I don't think the Soundcloud URL is the cause of the problem as I've always been able to sync my playlist with this URL

**Comment by rtomasello** [comment]
<img width="312" alt="capture" src="https://user-images.githubusercontent.com/24699269/27515079-50d29b48-5993-11e7-99a7-8d3e053769b8.PNG">

**Comment by erwinkramer** [comment]
I updated the app that will get you a more descriptive exception. Please update and paste the exception you are getting.

**Comment by rtomasello** [comment]
thank you very much for the update. the descriptive exception is now including the track id - https://soundcloud.com/mistatrick/caravan-palace-midnight-mista-trick-remix - which caused the issue. The format for this track is .aif and its size is over 40mb. I've now removed it from my likes and I've been able to download/sync my playlist with no problems as usual. thanks again for your help, really appreciated!
